### ISSUE TITLE
simplify multi-line strings

### ISSUE THREAD

**Issue by JimmyCushnie** [JimmyCushnie/SUCC#23]
Current way:
```
MultiLineString: """
    here is line one
    here is line two
    """
```
There's really no need for the triple quotes. Desired:
```
MultiLineString:
    here is line one
    here is line two
```
Complications with this: the parser needs to use a more advanced method of detecting a multi-line string, so that it knows not to fail when those lines don't have a `:` in them.

**Comment by pipe01** [comment]
I think the current syntax is fine, as it explicitly tells you that there's
a string in there

**Comment by JimmyCushnie** [comment]
That's what I thought as well when I designed it. Tonight I've finally actually started using files with multi-line strings, and my initial impression is that the triple quotes are annoying noise.

It's also worth noting that you'd still be able to tell for certain that there's a string there, since this is the only scenario in which lines of SUCC do not contain a `:` to separate the key from the data.

I'll keep working with these files and see if the triple quotes grow on me.
